### PR TITLE
chore(*): release 0.2.1

### PR DESCRIPTION
**Pull request by bungle**
### Summary

- fix(tests): fix tests to not be order dependent on multi-worker scenario (#46)
- chore(callback): remove unnecessary assert when the type is already checked (#50)
- chore(queue): rename DEFAULT_QUEUE_LEN to more meaningful MAX_QUEUE_PREALLOCATE (#52)
- chore(callback): split get_callback_list to two functions to avoid unnecessary table creation (#51)
- fix(worker): do not premature kill threads when not exiting (#47)
- fix(broker): do not premature kill threads when not exiting (#48)
- fix(worker): separate communication and event processing to different timers (#53) - caused by #47

### CODE REVIEW

File: lualib/resty/events/compat/init.lua [hunk 1]
```diff
@@ -14,7 +16,7 @@ local handlers = {}
 local _configured
 
 local _M = {
-    _VERSION = '0.1.0',
+    _VERSION = events._VERSION,
```
Comment (chronolaw) [inline lualib/resty/events/compat/init.lua:19]: Perhaps we should keep it as `0.1.0`, lua-resty-healthcheck may check this version number.

Comment (outsinre) [reply lualib/resty/events/compat/init.lua:19]: Seems the current version in the rest.events.init.lua is `0.2.0`. Would this be an inconsistency?

https://github.com/Kong/lua-resty-events/blob/7906f0c28e8081622bbb34d47da69bcf8abe944d/lualib/resty/events/init.lua#L16

Comment (chronolaw) [reply lualib/resty/events/compat/init.lua:19]: Currently, outside does not use `resty.events` directly, `resty.events.compat` is the real API in fact.

Comment (outsinre) [reply lualib/resty/events/compat/init.lua:19]: So, the result is: resty.events.compat as 0.1.0 and other components (the new lib) uses 0.2.0.

Comment (bungle) [reply lualib/resty/events/compat/init.lua]: Or, returned it back to `0.1.0`.

**Review by chronolaw — DISMISSED** [review]

**Review by outsinre — APPROVED** [review]

**Review by chronolaw — APPROVED** [review]

**Pull request merged**